__jest__/jest.transform.js

```diff
@@ -1,6 +1,9 @@
 // Custom Jest transform implementation that wraps babel-jest and injects our
 // babel presets, so we don't have to use .babelrc.
 
+/*eslint no-undef: "error"*/
+/*eslint-env node*/
+
 module.exports = require('babel-jest').createTransformer({
-  presets: ["env", "react", "stage-3"],
+  presets: ['@babel/preset-env', '@babel/preset-react', 'stage-3'],
```

Comment:
Do you need to fixed the stage-3 thing here as well?

Reply:
Fixed!